xapian-core/tests/api_weight.cc

```diff
@@ -134,6 +134,49 @@ DEFINE_TESTCASE(bm25weight5, backend) {
 }
 
 // Test exception for junk after serialised weight.
+DEFINE_TESTCASE(bm25plusweight3, !backend) {
```

Comment:
You are correct. Sorry, didn't really catch that earlier. I'm naming the testcases in `api_weight.cc` as `bm25plusweight1`, `bm25plusweight2` and so on.